app/models/assignment_repo/creator.rb

```diff
@@ -189,5 +190,69 @@ def suffixed_repo_name(repository_name, suffix_count)
       suffix = "-#{suffix_count}"
       repository_name.truncate(100 - suffix.length, omission: '') + suffix
     end
+
+    # Internal: Handles assignment repository setup
+    #
+    # configs_repo    - GitHubRepository containing the configuration files
+    # assignment_repo - GitHubRepository for which to perform the configuration
+    #                   setups
+    #
+    # Returns nothing
```

Comment:
Are there cases where something in here could fail and we would need to do something different?